ksqldb-engine/src/main/java/io/confluent/ksql/function/udf/string/Encode.java

```diff
@@ -70,7 +71,9 @@ public String encode(
       throw new KsqlFunctionException("Supported input and output encodings are: "
                                   + "hex, utf8, ascii and base64");
     }
-    return ENCODER_MAP.get(encodedString).apply(str);
+    return Optional.ofNullable(ENCODER_MAP.get(encodedString))
+        .orElseThrow(NullPointerException::new)
+        .apply(str);
```

Comment:
The new guava dependency makes specifies that the result of `get`-methods of immutable maps need be checked for nulls.